subo/command/build.go

```diff
@@ -90,26 +90,19 @@ func BuildCmd() *cobra.Command {
 				return errors.Wrap(err, "failed to BuildWithToolchain")
 			}
 
-			if shouldBundle {
-				if err := bdr.Bundle(); err != nil {
-					return errors.Wrap(err, "failed to Bundle")
-				}
+			pkgr := packager.New(&util.PrintLogger{})
+			pkgJobs := []packager.PackageJob{}
 
-				defer util.LogDone(fmt.Sprintf("bundle was created -> %s @ %s", bdr.Context.Bundle.Fullpath, bdr.Context.Directive.AppVersion))
+			if shouldBundle {
+				pkgJobs = append(pkgJobs, packager.NewBundlePackageJob())
 			}
 
 			if shouldDockerBuild && !bdr.Context.CwdIsRunnable {
```

Comment:
can a project use bundle and docker at the same time?

Reply:
Certainly! That's why the Packager method takes in a list of jobs.